Source/Core/DolphinQt2/Settings/GameCubePane.cpp

```diff
@@ -149,7 +149,7 @@ void GameCubePane::OnConfigPressed(int slot)
   {
   // Memory card
   case 2:
-    filter = tr("GameCube Memory Cards (*.raw,*.gcp)");
```

Comment:
Maybe you should consider adding a i18n comment so people don't touch the last part? Or move it out of the translatable string?

Reply:
We don't do that for the other strings that are used in file pickers, so it would be a bit weird to do it only here.

Reply:
Oh, okay. If it hasn't caused issues in the past, that looks fine to me.